tools/deployment/package-helm/values.yaml

```diff
@@ -0,0 +1,193 @@
+nameOverride: ""
+fullnameOverride: ""
+
+# Whether to allow sbin script accesses to services on the node host.
+allowSbinScripts: true
+
+storage:
+  # Base directory on host when using local PVs.
+  localPathBase: "/tmp/clp"
```

Comment:
maybe just a clpHome key at the root? or we enforce users to specify absolute paths in the clpConfig